math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -2313,10 +2313,9 @@ RVec<T> Take(const RVec<T> &v, const RVec<typename RVec<T>::size_type> &i, const
    return r;
 }
 
-
 /// Return first or last `n` elements of an RVec
 ///
-/// if `n > 0` and last elements if `n < 0`.
+/// Returns first elements if `n > 0` and last elements if `n < 0`.
```

Comment:
for consistency: 

```suggestion
/// Return first elements if `n > 0` and last elements if `n < 0`.
```

Reply:
Please update the docstring for consistency as suggested. Furthermore, there seems to be a repetition between the first line and this new line you are adding. Maybe merge the text in one line?